lib/commands/setup.js

```diff
@@ -1253,9 +1260,22 @@ SetupScreens.prototype.app = function app(callback) {
  * @param {Function} callback - Function to be called when the prompting finishes
  */
 SetupScreens.prototype.network = function app(callback) {
+	var defaultProxy = '';
+	if(this._config.get('cli.httpProxyServer')) {
+		defaultProxy = this._config.get('cli.httpProxyServer');
+	} else if(proxy.length > 0) {
+		var pI = 0, pL = proxy.length;
+		for(; pI < pL; pI++) {
```

Comment:
except that then pI and pL would be global variables. I admit the form is a bit unusual but it follows the pattern of other code in the project.